web/src/pages/DisplayTravelResults.jsx

```diff
@@ -65,7 +67,25 @@ function DisplayTravelResults() {
 		return [date, time];
 	};
 
-	// Function to fetch travel data from API
+	// Handle sorting when radio button changes
+	const handleOptionChange = (e) => {
+		setSelectedOption(e.target.value);
+	};
+
+	// Sort based on selectedOption
+	const sortedResults = [...processedResultsStorage].sort((a, b) => {
+		switch (selectedOption) {
+			case "min":
+				return a.minTravelTimeInMinute - b.minTravelTimeInMinute;
+			case "max":
+				return b.maxTravelTimeInMinute - a.maxTravelTimeInMinute;
+			case "minSlack":
+				return a.arrivalSlack - b.arrivalSlack;
+			default:
+				return 0; // Default, no sorting
```

Comment:
I will change the name to 'Meeting Time,' as it sounds more sensible. I thought it’s better to sort the results by Meeting Time by default since users expect to see the data in that order. If the rows are shuffled, it could be confusing for the organizer.